### PR TITLE
Season 2 Update Patch 3.1

### PR DESCRIPTION
**Pull request by CrazyIsEasy**
Updated challenge descriptions for the "R1 Shadowhunter"

- Fixed issue #15

### CODE REVIEW

File: src/data/camouflages.js [hunk 1]
```diff
@@ -61,7 +61,10 @@ export default [
       'Pistols': { default: '25 Longshots' },
       'Shotguns': { default: '50 Longshots' },
       'Sniper Rifles': { default: '50 Longshots' },
-      'Special': { default: 'Destroy 50 ground-based killstreaks' },
+      'Special': { 
+        default: 'Destroy 50 killstreaks or equipment',
+        'R1 Shadowhunter': '50 Longshots'
```
Comment (carlssonemil) [inline src/data/camouflages.js:66]: Shouldn't this be "50 Headshots" according to #15?

**Review by carlssonemil — CHANGES_REQUESTED** [review]
See comment on row 66 in camouflages.js file.

File: src/data/camouflages.js (hunk 1 above)
Comment (CrazyIsEasy) [reply src/data/camouflages.js]: Fixed!

**Review by carlssonemil — APPROVED** [review]

**Comment by carlssonemil** [comment]
Great job! 👍

**Pull request merged**